docs/documentation/form/file.html

```diff
@@ -739,6 +749,23 @@ <h2 class="subtitle">
       </p>
     </div>
 
+    {% include anchor.html name="Limitations" %}
+
+    <div class="message is-warning">
+      <div class="message-body">
+        <p>
+          Please be aware that browser's validity message, for example when value of input is <code>required</code>, will not be visible when using <code>.file</code>
```

Comment:
“the browser”
“when the input is required”
Move the “for example” clause at the end.